SmartDeviceLink-iOS/SmartDeviceLink/SDLRPCRequestFactory.h

```diff
@@ -143,6 +147,10 @@
 + (SDLRegisterAppInterface *)buildRegisterAppInterfaceWithAppName:(NSString *)appName languageDesired:(SDLLanguage *)laguageDesired appID:(NSString *)appID;
 //*****
 
+//***** Device Info ******
++(SDLDeviceInfo*) buildDeviceInfo;
```

Comment:
Correct, it should be private.  Also, it will eventually be incorporated as part of the ALM changes, so that the RAI RPC that will be sent automatically by ALM will have this info populated (instead of only when using the RAI factory method as above).